Added Michael to the list of authors

Calerid opened the pull request.

Calerid committed:
Added Michael to the list of authors

Calerid committed:
Added NT OS path for TWS API

tranceitionalMynd reviewed (changes requested):
Please add escape characters to the WIndows path: "C:\\TWS API\\source\\pythonclient"

Calerid committed:
Added version check for ver 9.73.2, please check for quality

Calerid committed:
Revert "Added version check for ver 9.73.2, please check for quality"

This reverts commit 21ba5c6642b16a614c18e67f6dd4e1b862c0ccf7.

Calerid committed:
Escapes in path check, added version check.

added escapes to back slashes in alternate_path for nt operating systems.
Also added TWS API version check. Please see if this is an acceptable method.

tranceitionalMynd reviewed (changes requested):
Thanks for fixing the double backslashes.  The convention for the failed syntax checks is:

sys.stderr.write(ERROR_MESSAGE + "\n")
sys.exit(1)

This prints the error message (on standard error) and exits the program with a failure code (zero indicates success).

We want to avoid printing a backtrace of exceptions to the user since we already know what the problem is.

tranceitionalMynd commented on spy-reaper/spy-reaper.py:
```diff
@@ -18,6 +20,13 @@
+tws_req_ver = "9.73.2"
+
+if tws_api_ver != tws_req_ver:
+    raise sys.stderr.write("twsapi version '{0}' required, version installed is '{1}'".format(tws_req_ver, tws_api_ver))
```
Please follow the error handling convention for system sanity checks:
sys.stderr.write("ERROR MESSAGE\n")
sys.exit(1)

This will terminate the program with a non-zero exit code indicating failure.

Using "raise" without an "except" to catch it will print a full backtrace of exceptions which is hard for the user to understand.

NOTE: We may decide to do error-handling differently elsewhere in the project, but for system sanity checks this is the current convention.

Calerid committed:
Made suggested fixes to code, conformed version to prior convention.
